tools/diffdeb

```diff
@@ -10,20 +10,36 @@
 # OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
 # specific language governing permissions and limitations under the License.
 
+verbose=$1
+if [ "$verbose" == "-v" ]; then
+    shift
+else
+    verbose=""
+fi
 a=$1
 b=$2
 if [ "$b" = "" ]; then
-    echo "Usage: $0 <file1.deb> <file2.deb>"
+    echo "Usage: $0 [-v] <file1.deb> <file2.deb>"
```

Comment:
Please update the help string ("identical in all aspects..." etc.)

Reply:
committed to dev as 54f456f1